### PR TITLE
fix(condo): DOMA-5956 offline status change history

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**

### CODE REVIEW

File: apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js [hunk 1]
```diff
@@ -282,8 +286,41 @@ async function classifyTicket (context, details) {
...
 
+const STATUS_RELATED_TICKET_FIELDS = ['status', 'statusUpdatedAt', 'deferredUntil']
+
+const createOfflineTicketStatusChange = async ({ schema, operation, context, existingItem, resolvedData }) => {
+    await storeChangesIfUpdated(
+        pick(schema.fields, ['status']),
+        createTicketChange({ isOfflineChange: true, actualCreationDate: resolvedData.statusUpdatedAt }),
+        ticketChangeDisplayNameResolversForSingleRelations,
+        relatedManyToManyResolvers,
+    )({ operation, existingItem: { id: existingItem.id }, context, updatedItem: { sender: existingItem.sender, status: resolvedData.status } })
```
Comment (Alllex202) [inline apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js:297]: `deferredUntil` won't be set if payload have it?

File: apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 const dayjs = require('dayjs')
-const { isUndefined, isEmpty, get } = require('lodash')
+const { isUndefined, isEmpty, get, pick } = require('lodash')
+const omit = require('lodash/omit')
```
Comment (Alllex202) [inline apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js:3]: should to merge

File: apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js (hunk 1 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js]: fixed

File: apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js [hunk 3]
```diff
@@ -282,8 +285,46 @@ async function classifyTicket (context, details) {
     })
 }
 
+const STATUS_RELATED_TICKET_FIELDS = ['status', 'statusUpdatedAt', 'deferredUntil']
+
+const createOfflineTicketStatusChange = async ({ schema, operation, context, existingItem, resolvedData }) => {
+    await storeChangesIfUpdated(
+        pick(schema.fields, ['status']),
```
Comment (Alllex202) [inline apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js:292]: If you update `deferredUntil` then you should pick this field too for these changes to be saved in TicketChange.

File: apps/condo/domains/ticket/schema/Ticket.test.js
Comment (Alllex202) [inline apps/condo/domains/ticket/schema/Ticket.test.js:1]: Please add some tests for case with updating `deferredUntil` from offline

**Review by Alllex202 — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/ticket/schema/TicketMultipleUpdateService.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+const get = require('lodash/get')
+
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/ticket/access/TicketMultipleUpdateService')
+const { DEFAULT_STATUS_TRANSITIONS } = require('@condo/domains/ticket/constants/statusTransitions')
+
+const { Organization } = require('../../organization/utils/serverSchema')
+const { Ticket } = require('../utils/serverSchema')
```
Comment (Alllex202) [inline apps/condo/domains/ticket/schema/TicketMultipleUpdateService.js:13]: Fix imports

File: apps/condo/domains/ticket/schema/TicketMultipleUpdateService.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const access = require('@condo/domains/ticket/access/TicketMultipleUpdateService')
+const { DEFAULT_STATUS_TRANSITIONS } = require('@condo/domains/ticket/constants/statusTransitions')
+const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+
+
+const TicketMultipleUpdateService = new GQLCustomSchema('TicketMultipleUpdateService', {
```
Comment (Alllex202) [inline apps/condo/domains/ticket/schema/TicketMultipleUpdateService.js:15]: pls add `schemaDoc`

File: apps/condo/domains/ticket/utils/serverSchema/TicketChange.js [hunk 6]
```diff
@@ -12,12 +13,23 @@ const { TicketChange } = require('./index')
...
         sender: updatedItem.sender,
         ticket: { connect: { id: existingItem.id } },
         ...fieldsChanges,
     }
+
+    if (newItem.statusUpdatedAt) {
+        const statusUpdatedAtDateTime = dayjs(newItem.statusUpdatedAt)
+        const statusUpdatedAtDiffFromNow = Math.abs(dayjs().diff(dayjs(statusUpdatedAtDateTime), 'second'))
+
+        if (statusUpdatedAtDiffFromNow > 10) {
```
Comment (Alllex202) [inline apps/condo/domains/ticket/utils/serverSchema/TicketChange.js:28]: move `10` to const

**Review by Alllex202 — COMMENTED** [review]
Write a little about what it is for and how it works

File: apps/condo/domains/ticket/schema/TicketMultipleUpdateService.js (hunk 5 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/TicketMultipleUpdateService.js:15]: added

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**